src/app/button-group/angular/mixed-buttons/mixed-button-group.html

```diff
@@ -0,0 +1,29 @@
+<!--
+  ~ Copyright (c) 2016 VMware, Inc. All Rights Reserved.
```

Comment:
As we add these in, I think they are supposed to say `2016-2017`. I guess it is not a big deal unless someone else chimes in.